dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ParseConfigUtils.java

```diff
@@ -130,11 +134,23 @@ private void doRefreshItems() {
   }
 
   private Map<String, Object> mergeDimensionItems(Map<String, Map<String, Object>> items) {
-    Map<String, Object> flatMap = new HashMap<>();
-    return items.values().stream().reduce(flatMap, (result, item) -> {
-      result.putAll(item);
-      return result;
-    });
+	Map<String, Object> flatMap = new HashMap<>();
```

Comment:
Your code format style is wrong, please use the format file under https://github.com/apache/servicecomb-java-chassis/tree/master/etc